modules/data.land/R/IC_BADM_Utilities.R

```diff
@@ -0,0 +1,296 @@
+
+#Reading the data and writing the netcdf
+#' Read.IC.info.BADM
+#'
+#' @param lat numeric latitude
+#' @param long numeric longitude
+#'
+#' @description This function returns a dataframe of plant biomass, root and soil carbon for a set of lat and long coordinates.
+#' This function first finds the level1 and level2 ecoregions  for the given coordinates, and then tries to filter BADM database for those eco-regions. 
+#' If no data found in the BADM database for the given lat/longs eco-regions, then all the data in the database will be used to return the initial condition.
+#' All the variables are also converted to kg/m^2. 
+#' @return a dataframe with 8 columns of Site, Variable, Date, Organ, PlantWIni (Initial plant biomass, type of biomass can be found in the Variable and Organ), SoilIni (which shows the initial soil C), LitterIni, RootIni.
+
+#' @export
+#'
+#' @examples
+Read.IC.info.BADM <-function(lat, long){
```

Comment:
Also the way you have this setup right now, the entire function is the example.  You need to either 1) remove the example from the doc, or 2) provide an example before the function, such as:

badm_test <- Read.IC.info.BADM(45.805925,-90.07961)

but you probably want to place it in the dontrun{} tags since this could break travis if it can't reach the portal for the badm files